pkg/storage/mvcc_stats_test.go

```diff
@@ -1609,7 +1614,25 @@ func (s *randomTest) step(t *testing.T) {
 	actName := s.actionNames[s.rng.Intn(len(s.actionNames))]
 
 	preTxn := s.Txn
-	info := s.actions[actName](&s.state)
+	s.batch = s.internal.eng.NewBatch()
+	*s.MSDelta = enginepb.MVCCStats{}
+	ok, info := s.actions[actName](&s.state)
+	if ok {
+		s.internal.ms.Add(*s.MSDelta)
+		require.NoError(t, s.batch.Commit(false /* sync */))
+		if *s.MSDelta != (enginepb.MVCCStats{}) {
```

Comment:
This isn't comparing equality or computing a diff, this is just trying to print the nonzero fields. But I switched `assertEqImpl` to use `require.Equal`, and removed the JSON encoding here since it's not useful any more now that I've removed the `,omitempty` commit.